src/components/about/About.jsx

```diff
@@ -6,43 +6,43 @@ const ABOUT_SECTION_STYLE = {
 
 const About = () => (
   <div id="about-container">
-    <div className="section-headers">
-      ABOUT
-    </div>
+    <div className="section-headers">ABOUT</div>
     <div className="about-section">
-      <div
-        className="about-subheader"
-        style={ABOUT_SECTION_STYLE}
-      >
+      <div className="about-subheader" style={ABOUT_SECTION_STYLE}>
         About Kubiquity
       </div>
       <p className="about-text">
-        Kubiquity is a Kubernetes error logging and tracking tool developed and launched in 2021. Designed for production-level applications using Kubernetes to orchestrate containers, Kubiquity provides metrics on all pods' memory use to help developers prevent OOM kill errors; additionally, Kubiquity logs Kubernetes events to highlight any errors and warnings that originate within monitored clusters.
```

Comment:
My preference would be to keep this on  one line—possibly save this text into a variable add the eslint ignore comment, or see if we can add in that comment via JSX. not a huge deal, but if we wanted to make changes to this text it's easier to do it when it's one line instead of adjusting the line breaks.